### PR TITLE
Make opml import favor xmlUrl over htmlUrl

### PR DESCRIPTION
**Pull request by kuba-orlik**
When importing this opml file: 

```
<?xml version='1.0' encoding='UTF-8' standalone='no' ?>
<opml version="2.0">
  <head>
    <title>AntennaPod Subscriptions</title>
    <dateCreated>17 Apr 17 12:52:03 +0200</dateCreated>
  </head>
  <body>
    <outline text="Developer Tea" title="Developer Tea" type="rss" xmlUrl="http://feeds.feedburner.com/DeveloperTea" htmlUrl="http://www.developertea.com" />
  </body>
</opml>
```

The resulting serverlist looked like so: 

```
http://www.developertea.com OPML_Import_2017-04-17 Developer Tea
```

The podcast download didn't work, because the url in the serverlist is the htmlUrl, not the xmlUrl which leads to the rss/atom feed. 

This simple change in a regexp fixes this.

### CODE REVIEW

File: podget [hunk 1]
```diff
@@ -1142,7 +1142,8 @@ PLAYLIST_ConvertToASX() {
 
      # Removing unix2dos dependency. Converting to sed statement with in-place editing of the file in question.
      # ctrl-v ctrl-m for windows line end.
-     sed -i 's/$//' "${DIR_LIBRARY}"/"${ASX_PLAYLISTNAME}"
+     sed -i 's/$/
+/' "${DIR_LIBRARY}"/"${ASX_PLAYLISTNAME}"
```
Comment (dvehrs) [inline podget]: Is this change needed?  It's been a little while since I tested dos file to unix conversions but as I recall the ctrl-M was the way to go....

File: podget [hunk 2]
```diff
@@ -336,7 +336,7 @@ ASX_PLAYLIST=0
 # to normalize files to have the same volume.
 #
 # A period (.) will automatically be added between the filename and tag like so:
-#       fileanme.mp3.newtag
+#       filename.mp3.newtag
```
Comment (dvehrs) [inline podget]: Nice catch.  If this were by itself it would be easy to merge.

File: podget [hunk 3]
```diff
@@ -2823,7 +2823,7 @@ if [[ -n ${IMPORT_OPML+set} ]]; then
             fi
 
             new_label=$(echo "${data}" | sed -n -e 's/.*text="\([^"]\+\)".*/\1/Ip' | sed -e 's/^\s*[0-9]\+\.\s\+//' -e "s/[:;'\".,!/?<>\\|]//g")
-            new_url=$(echo "${data}" | sed -n -e 's/.*[xml]*url="\([^"]\+\)".*/\1/Ip' | sed -e 's/ /%20/g')
+            new_url=$(echo "${data}" | sed -n -e 's/.*\(xml\|\W\)url="\([^"]\+\)".*/\2/Ip' | sed -e 's/ /%20/g')
```
Comment (dvehrs) [inline podget]: It's been a while since I checked the OPML syntax but this looks good.

File: podget (hunk 1 above)
Comment (kuba-orlik) [inline podget]: Whops, it seems it was incidental. Will remove it.
